android/app/src/main/java/app/organicmaps/widget/menu/NavMenu.java

```diff
@@ -213,20 +206,6 @@ private void updateSpeedView(@NonNull RoutingInfo info)
       return;
 
     Pair<String, String> speedAndUnits = StringUtils.nativeFormatSpeedAndUnits(last.getSpeed());
-    mSpeedValue.setText(speedAndUnits.first);
-
-    if (info.speedLimitMps > 0.0 && last.getSpeed() > info.speedLimitMps)
-    {
-      if (info.isSpeedCamLimitExceeded())
-        mSpeedValue.setTextColor(ContextCompat.getColor(mActivity, R.color.white_primary));
-      else
-        mSpeedValue.setTextColor(ContextCompat.getColor(mActivity, R.color.base_red));
-    }
-    else
-      mSpeedValue.setTextColor(ThemeUtils.getColor(mActivity, android.R.attr.textColorPrimary));
-
-    mSpeedUnits.setText(speedAndUnits.second);
-    mSpeedViewContainer.setActivated(info.isSpeedCamLimitExceeded());
```

Comment:
Nobody approved this removal yet :)
I think it's better to make it in separate PR

Reply:
> Nobody approved this removal yet :)
> I think it's better to make it in separate PR

@AndrewShkrob, @TobiPeterG: I've launched [PR #9955](https://github.com/organicmaps/organicmaps/pull/9955) for the removal of the speed display in the navigation panel, so we can discuss there the rearrange of the items in the navigation panel while the new Speed widget is implemented.